### DISCUSSION TITLE
Next catalog, no new one-offs on TNS after 2019-09-17

### DISCUSSION THREAD

**Discussion by HarshiniParanjape** [chime-frb-open-data/community#49] [Q&A] [answered]
When would the next catalog be released? Also, from the TNS data for CHIME, I could not find any new one-offs being reported after 2020-9-17,  is that consistent ? In fact, it has only 3 one-offs for the period 2019-09-01 to 2020-09-01. 
Thank you!

**Comment by fabianschussler** [comment]
As a follow-up: what is the strategy between sending real-time alerts and submission to TNS? Is the latter happening only during catalog releases?

**Reply by 0xCoto** [reply]
I'd also be interested to know. My primary source for checking for new FRB events from CHIME and other instruments has been the TNS. I've even set up an [action](https://github.com/HeRTA/FRBSTATS/actions/workflows/tns_compare.yml) to check for updates, since the official page doesn't support such utility, so I'd support submitting new events to the TNS as soon as possible.

**Comment by andrewzwaniga** [comment, marked as answer]
Hi folks,

Thanks for your inquiries. I will try to answer these questions to the best of my ability.

Regarding VOEvents and the TNS, we are tentatively working on an automated system that uploads FRBs to the TNS following internal human verification of the CHIME/FRB event data. The process would look something like this:

FRB detection --> detection-type VOEvent --> internal CHIME/FRB human verification --> TNS submission --> update-type VOEvent

Note that the initial detection-type VOEvent is delivered before human verification. Due to the non-zero false positive rate of the real-time FRB detection pipeline, some of these VOEvents may not be true FRBs. But following human verification, the idea is to produce an update-type VOEvent.

The update VOEvent would be a short message containing a reference to the original detection VOEvent (by IVORN), containing both the internal CHIME/FRB event number and the TNS name. Ideally this system would come online in the next few months.

At this time I cannot confirm when the next catalog will be released, nor can I confirm when any additional post-catalog 1 FRBs will be released in bulk on the TNS. However, I invite you to stay tuned, and I may have an update on this matter in the coming weeks.

Best regards,
Andrew
